src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java

```diff
@@ -145,6 +155,33 @@ public TableDescription deleteTable(String tableName) {
         return tableDescription;
     }
 
+    @Override
+    public Map<TenantTable, CreateTableRequest> getAllMtTables() {
```

Comment:
Should we considering returning a Stream or something similar from this method? I could imagine this collection getting too large to load into memory.

Reply:
addressed in https://github.com/salesforce/mt-dynamo/pull/449